browser/gui/app.cpp

```diff
@@ -156,6 +157,35 @@ void App::set_scale(unsigned scale) {
     engine_.set_layout_width(windowSize.x / scale_);
 }
 
+void App::sticky_windows(sf::Event::SizeEvent size) {
+    const char *names[] = {
+        "Navigation",
+        "HTTP Response",
+        "DOM",
+        "Stylesheet",
+        "Layout",
+        NULL,
+    };
+    
+    int delta = k_width - size.width;
+    for (uint8_t i = 0;; ++i) {
```

Comment:
`std::uint8_t`. There's no guarantee that <cstdint> places things in the global namespace, and once we start using modules, it's guaranteed that this won't be in the global namespace.